doc/source/getting_started/10min.rst

```diff
@@ -713,6 +713,9 @@ See the :ref:`Plotting <visualization>` docs.
 
 .. ipython:: python
 
+   # register converters to display dates in plots
+   pd.plotting.register_matplotlib_converters()
```

Comment:
This should not be added, the warning in the log is indicating a recent regression, see https://github.com/pandas-dev/pandas/pull/26770